app/javascript/custom/modal.js

```diff
@@ -61,6 +60,13 @@ $(document).ready(() => {
     return $('.modal').find('.modal-errors').html(error_messages);
   });
 
+  //opens modal after ajax:succes
+  $("form#new-password-user")
+  .on("ajax:success", function(event) {
+    $(this).parents('.modal').modal('hide');
+    $("#password-instructions-modal").modal("show");
+  })
```

Comment:
In these cases if you move `.on` to the next line, please add indents like:
```js
$("form#new-password-user")
  .on("ajax:success", function(event) {
    $(this).parents('.modal').modal('hide');
    $("#password-instructions-modal").modal("show");
  })
```
It be more readable